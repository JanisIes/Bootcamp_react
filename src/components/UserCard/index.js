import React, { Component } from "react";
import "./index.css";
import image from "../../media/image-placeholder.svg";

class UserCard extends Component {
  render() {
    return (
      <div className="user-wrapper">
        <div className="row">
          <div className="col-md-2">
            <img className="user-image" src={image} />
          </div>
          <div className="col-md-10">
            <div className="user-heading">Name</div>
            <div className="user-description">Desc</div>
          </div>
        </div>
      </div>
    );
  }
}

export default UserCard;
