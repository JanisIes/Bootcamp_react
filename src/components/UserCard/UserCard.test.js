import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UserCard from "./index";

describe("testing UserCard", () => {
  const wrapper = shallow(<UserCard />);
  it("renders snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
