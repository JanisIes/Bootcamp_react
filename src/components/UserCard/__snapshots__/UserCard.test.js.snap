// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing UserCard renders snapshot 1`] = `
<div
  className="user-wrapper"
>
  <div
    className="row"
  >
    <div
      className="col-md-2"
    >
      <img
        className="user-image"
        src="image-placeholder.svg"
      />
    </div>
    <div
      className="col-md-10"
    >
      <div
        className="user-heading"
      >
        name
      </div>
      <div
        className="user-description"
      >
        Desc
      </div>
    </div>
  </div>
</div>
`;
